views/partials/dashboards/remove-form.hbs

```diff
@@ -110,9 +110,10 @@
             <div class="remove-dashboard-form-error --loc"></div>
           </div>
           <div class="input-group">
-            <label for="remove-dashboard-form-year" class="input-group-label field--full required">{{ getRemoveString "remove-dash-form-birth-year-label"}}
+            <div class="flx flx-cntr">
+              <label for="remove-dashboard-form-year" class="input-group-label required">{{ getRemoveString "remove-dash-form-birth-year-label"}}</label>
               {{> removal/removal-question-trigger modal="modal-birth-year" alt="Why we require birth year"}}
```

Comment:
Should the `alt` be localized?

Reply:
I'll put it in strings to be consistent - good catch! (none of the `getRemoveString` function calls actually get localized, but it keeps that possibility open).